fossunited/fossunited/doctype/foss_event_cfp_submission/foss_event_cfp_submission.py

```
@@ -417,3 +418,95 @@ def notify_team_approved_proposal_withdrawn(self):
+            return
+        speaker_emails = [s.email for s in self.speakers if s.email]
+        try:
+            frappe.sendmail(
+                recipients=self.email,
+                cc=speaker_emails,
+                subject=f"{subject_prefix} {self.event_name}",
+                message=build_review_message(self, review, change_type, old_review),
+                reference_doctype=PROPOSAL,
+                reference_name=self.name,
+            )
+        except Exception:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)